runtime/rasdump/javadump.cpp

```diff
@@ -5291,7 +5292,38 @@ JavaCoreDumpWriter::writeCPUinfo(void)
 	_OutputStream.writeCharacters(
 			"2CITARGETCPU   Target CPUs: ");
 	_OutputStream.writeInteger(target, "%i\n");
+	
+	int BUFF_SIZE = 400;
+	char buff[BUFF_SIZE];
```

Comment:
Are we able to fold `BUFF_SIZE` into `buff` and replace uses of `BUFF_SIZE` with `sizeof(buff)`?